plugins/generic/usageStats/PKPUsageStatsPlugin.inc.php

```diff
@@ -186,6 +185,10 @@ function callbackLoadCategory($hookName, $args) {
 			$this->import('UsageStatsReportPlugin');
 			$plugin = new UsageStatsReportPlugin();
 		}
+		if ($category ==  'blocks') {
+			$this->import('UsageStatsOptoutBlockPlugin');
+			$plugin = new UsageStatsOptoutBlockPlugin($this->getName());
```

Comment:
Ah, I could check here if the option is chosen, right? Thanks Bruno!!! I will then change those two things (the typo too) and let you know...